SPECS/tcl/tcl.spec

```
@@ -1,16 +1,14 @@
+%define majorver 8.6
```

Nit: might be worth replacing with macros, which automatically extract this value. Example from `kf5-kconfig.spec`:

```suggestion
%define majmin %(echo %{version} | cut -d. -f1-2)
```

---

Great suggestion. Fixed in newest version.